base_partner_merge/base_partner_merge.py

```diff
@@ -381,12 +381,15 @@ def _merge(self, cr, uid, partner_ids, dst_partner=None, context=None):
             src_partners = ordered_partners[:-1]
         _logger.info("dst_partner: %s", dst_partner.id)
 
-        call_it = lambda function: function(cr, uid, src_partners,
-                                            dst_partner, context=context)
-
-        call_it(self._update_foreign_keys)
-        call_it(self._update_reference_fields)
-        call_it(self._update_values)
+        self._update_foreign_keys(
+            cr, uid, src_partners, dst_partner, context=context
+        )
```

Comment:
Why to you put the last ')' on a new line? Sometimes it's at the end of  line sometimes on a new line...  (I know it's a kind a nit-picking :smirk: )